### ISSUE TITLE
Certain PR checks always fail

### ISSUE THREAD

**Issue by lwcorp** [phpList/phplist3#997]
If someone tries to send a PR to your code, two commit checks always fail.

**Steps to reproduce**
1. Edit a file or create a new one.
1. Commit it.
1. Send a PR.

**Expected result**
Pass all checks.

**Actual outcome**
The second and third checks from the end (7.2 and 7.3) always fail.
You can see them for further details in every [open PR](https://github.com/phpList/phplist3/pulls). The important part is it has nothing to do with the PR, but looks like it is.

![image](https://github.com/phpList/phplist3/assets/1773306/fdcf6d03-627f-4e5a-a1c3-175f7b252667)

**Comment by michield** [comment]
yes, that's why they are marked to "allow to fail". They fail on composer dependencies. We'll take them out soon

**Comment by lwcorp** [comment]
Where does it show "allow to fail"?

**Comment by michield** [comment]
![image](https://github.com/phpList/phplist3/assets/569843/50eabcd4-f371-48d1-bb10-f52c69a33db7)

**Comment by lwcorp** [comment]
Do only admins see it?

**Comment by michield** [comment]
Ah, don't know. Can you see it here? https://github.com/phpList/phplist3/actions/runs/6760239013

Also, the overall run has a blue tick, as opposed to a red one when all of them fail https://github.com/phpList/phplist3/actions

**Comment by lwcorp** [comment]
Some actions there have a fail icon and some "just" a warning.

In #994 for example to know it's okay not to pass some of these tests?

**Comment by michield** [comment]
This is now ready for review in #1004
